site/_posts/2022-03-10-announcing-fusionauth-1-35.md

```diff
@@ -0,0 +1,100 @@
+---
+layout: blog-post
+title: Announcing FusionAuth 1.35
+description: This release includes bug fixes, internal updates, and support for API calls in FusionAuth lambdas.
+author: Dan Moore
+image: blogs/release-1-35/product-update-fusionauth-1-35.png
+category: blog
+tags: tbd topic-troubleshooting feature-advanced-threat-detection
+excerpt_separator: "<!--more-->"
+---
+
+We're excited to announce the release of version 1.35 of FusionAuth. This version shipped Mar 9, 2022. 1.35 includes bug fixes, internal updates, and support for API calls in FusionAuth lambdas.
+
+<!--more-->
+
+This release contained features, enhancements, and bug fixes. Please see the [release notes](/docs/v1/tech/release-notes#version-1-35-0) for a full breakdown of the changes between 1.34 and 1.35. 
+
+There are a few improvements that I wanted to call out specifically.
+
+## API calls within lambdas
+
+This is an [oft-requested feature](https://github.com/FusionAuth/fusionauth-issues/issues/267) which allows for complex integrations by allowing a lambda to retrieve data from external URLs.
+
+As a reminder, FusionAuth lambdas are snippets of JavaScript executed at various points in the auth lifecycle, including:
+
+* Before a JWT is generated when a user has logged in.
+* After a response is received from an Identity Provider such as Google or a SAML IdP.
+* Before a JWT is issued in a Client Credentials grant.
+* After user data has been retrieved from an LDAP server using a Connector.
+
+In all of these scenarios, integration with additional data sources or logic may be required or useful. This includes:
+
+* Adding custom claims based on business logic specific to the user and application, encapsulated by an API.
+* Calling IdP endpoints to retrieve additional data, such as the Microsoft Graph API endpoints.
+* Augmenting a token with information from FusionAuth, such as the name of any groups of which a user is a member.
```

Comment:
By this I think you mean calling our own API from lambda, should we say that more specifically, and say, anything you ca get from a FusionAuth API, you can now get in a lambda.